cypress/fixtures/items.js

```diff
@@ -254,3 +255,5 @@ export const GRAASPER_ITEMS = [
     categories: [{ categoryId: SAMPLE_CATEGORIES[1].id }],
   },
 ];
+
+export const getNumberOfOwnPublishedItems = (memberId) => PUBLISHED_ITEMS.filter(({path, creator}) => path.split('.').length === 1 && creator === memberId).length
```

Comment:
`path.split('.').length===1` -> Wouldn't it be easier to test if it contains `.` ?